### PR TITLE
Default textBaseline to alphabetic in Flex

### PR DESCRIPTION
**Pull request by tvolkert**
## Description

Per the discussion in https://github.com/flutter/flutter/issues/58053, this follows-up https://github.com/flutter/flutter/pull/60586 by defaulting `textBaseline` in `Flex` and `Row` as well.

## Related Issues

https://github.com/flutter/flutter/issues/58053

## Tests

n/a

## Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: https://flutter.dev/go/breaking-changes-template *Replace this with a link to a pull request that adds the migration guide to https://flutter.dev/docs/release/breaking-changes*

### CODE REVIEW

**Comment by goderbauer** [comment]
Looks like cirrus is not happy with this.

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -3863,7 +3863,7 @@ class Flex extends MultiChildRenderObjectWidget {
     this.crossAxisAlignment = CrossAxisAlignment.center,
     this.textDirection,
     this.verticalDirection = VerticalDirection.down,
-    this.textBaseline,
+    this.textBaseline = TextBaseline.alphabetic,
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/basic.dart:3866]: Can you update the documentation of this property to include what the default value is?

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/widgets/basic.dart (hunk 1 above)
Comment (tvolkert) [reply packages/flutter/lib/src/widgets/basic.dart:3866]: Done

**Comment by Hixie** [comment]
We should revert this. We have no way to know that the correct baseline is alphabetic.

**Pull request merged**